spec/services/concerns/filter_validatable_spec.rb

```diff
@@ -1,42 +0,0 @@
-class FilterValidatableTestsController < API::BaseController
```

Comment:
@leandroalemao discussed this with @ethax-ross as I asked for this spec in your pr, we agreed in the end that controller concerns can be tested as part of the shared contexts in request specs, happy to chat later today to agree